scene/2d/animated_sprite_2d.cpp

```diff
@@ -382,6 +382,7 @@ bool AnimatedSprite2D::_is_playing() const {
 }
 
 void AnimatedSprite2D::play(const StringName &p_animation, const bool p_backwards) {
+	ERR_FAIL_NULL_MSG(frames, "SpriteFrames reference is invalid (NULL) in AnimatedSprite2D::play()");
```

Comment:
The actual error will already include a mention of `AnimatedSprite2D::play()` and `frames` being `NULL`, so it doesn't need to be repeated. I'd suggest instead:
```suggestion
	ERR_FAIL_NULL_MSG(frames, "Can't play AnimatedSprite2D without a valid SpriteFrames resource.");
```

Reply:
Can't see null pointer is included in the default error message but seems to be easier to understand anyway. 

![Capture d’écran 2021-03-17 150211](https://user-images.githubusercontent.com/3649998/111479981-c7ca4000-8731-11eb-8caf-90442b4982ac.png)

Done.

Reply:
It should be included in Godot's debugger when you click the error ("C++ error").